docs/specs/account-management-api.md

```diff
@@ -0,0 +1,861 @@
+# Account Management API
+
+Account Management API is an API for account management on behalf of the end user. Account Management API requires a authenticated session, either by cookie or the HTTP header Authorization.
+
+Account Management API supports the following operations:
+
+- [Manage identification methods](#manage-identification-methods)
+  - [List identification methods](#list-identification-methods)
+  - [Add a new email, with verification](#add-a-new-email-with-verification)
+  - [Add a new phone number, with verification](#add-a-new-phone-number-with-verification)
+  - [Add a new username](#add-a-new-username)
+  - [Add an OAuth provider account, with authorization code flow](#add-an-oauth-provider-account-with-authorization-code-flow)
+  - [Add a biometric](#add-a-biometric)
+  - Add a passkey
+  - [Remove an email](#remove-an-email)
+  - [Remove a phone number](#remove-a-phone-number)
+  - [Remove a username](#remove-a-username)
+  - [Remove an OAuth provider account](#remove-an-oauth-provider-account)
+  - [Remove a biometric](#remove-a-biometric)
+  - [Remove a passkey](#remove-a-passkey)
+  - [Update an email, with verification](#update-an-email-with-verification)
+  - [Update a phone number, with verification](#update-a-phone-number-with-verification)
+  - [Update a username](#update-a-username)
+- Manage authentication methods
+  - List authentication methods, and recovery codes.
+  - Change the primary password
+  - Change the secondary password
+  - Remove the secondary password
+  - Add a new TOTP authenticator
+  - Remove a TOTP authenticator
+  - Add a new OOB-OTP authenticator, with verification.
+  - Remove a OOB-OTP authenticator
+  - Re-generate recovery codes
+- Manage user profile
+  - Update simple standard attributes
+  - Update custom attributes
+  - Add user profile picture
+  - Replace user profile picture
+  - Remove user profile picture
+- Manage sessions
+  - List sessions
+  - Revoke a session
+  - Terminate all other sessions
+- [Auxiliary operations](#auxiliary-operations)
+  - [Verify OTP](#verify-otp)
+  - [Resend OTP](#resend-otp)
+
+## Manage identification methods
+
+### List identification methods
+
+`GET /api/v1/account/identification`
+
+Response
+
+```json
+{
+  "result": {
+    "identification_methods": [
```

Comment:
I am not sure if identification is a countable noun. Some dictionaries I found on Google say it could be countable. 

In authflow, it is used as

```
{
  "identification": "email"
}
```

Let me change it to 

```
{
  "identifications": [
    {
      "identification": "email"
    }
  ]
}
```